platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -104,7 +104,7 @@
 import com.swirlds.platform.gossip.GossipFactory;
 import com.swirlds.platform.gossip.IntakeEventCounter;
 import com.swirlds.platform.gossip.NoOpIntakeEventCounter;
-import com.swirlds.platform.gossip.shadowgraph.ShadowGraph;
+import com.swirlds.platform.gossip.shadowgraph.Shadowgraph;
```

Comment:
Why the change from CamelCase?

Reply:
We always use "shadowgraph" when speaking as if it were a single word, and the camel case always seemed jarring to me. Just an opportunistic cleanup, not strictly required for the core objective of this change.

Reply:
We also have `ShadowEvent` and there are other files called `ShadowConfigurationAccessors`.    I see it as two words since there are other `Shadow` things.   Seems like a highly individual perception based on how language operates within them.  You've worked with it long enough that it has become one word to you.  :-)